Package.swift

```diff
@@ -0,0 +1,78 @@
+// swift-tools-version:4.2
+
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2019 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+
+import PackageDescription
+
+let package = Package(
+    name: "swift-tools-support-core",
+    products: [
+        .library(
+            name: "SwiftToolsSupport",
+            type: .dynamic,
+            targets: ["TSCBasic", "TSCUtility"]),
+        .library(
+            name: "SwiftToolsSupport-auto",
+            targets: ["TSCBasic", "TSCUtility"]),
+    ],
+    dependencies: [
+        
+    ],
+    targets: [
+        
+        // MARK: Tools support core targets
+        
+        .target(
+            /** Shim target to import missing C headers in Darwin and Glibc modulemap. */
+            name: "TSCclibc",
```

Comment:
Can we call this _TSCClibc or some such?

Reply:
Same as below, I copied the definition from SwiftPM. Are you worried about the visibility due to the TSC prefix although it's kinda useless outside?

Reply:
I just didn't like the naming convention itself, the lowercase clibc. Not important, you can disregard.